src/main/scala/com/typesafe/sbteclipse/core/EclipseAutoPlugin.scala

```diff
@@ -0,0 +1,25 @@
+package com.typesafe.sbteclipse.plugin
+
+import sbt.{Setting, AutoPlugin}
+
+import com.typesafe.sbteclipse.core.{EclipsePlugin=>EclipseCorePlugin}
+
+import scala.xml.Node
+
+/** Migrates eclipse to be an autoplugin. */
+object EclipsePlugin extends AutoPlugin {
+  override def requires = sbt.plugins.JvmPlugin
+  override def trigger = allRequirements
+  // Auto import semantics users expect today.
```

Comment:
True. I've tried to preserve enough that we don't get massive breakage in existing builds, and the tests don't change.   We can be more aggressive if we want, but this should get us over the hump of a usable release.